README.md

```diff
@@ -1,4 +1,208 @@
 eos-knowledge-lib
 =================
 
-GUI library for knowledge apps
+Welcome to **eos-knowledge-lib**!
+This is Endless' Javascript library for building offline content apps using modular UI blocks and declarative UI files.
+
+This project uses the GNOME technology stack, including GTK and WebKit for UI, and GJS for its Javascript interpreter.
+
+Building
+--------
+We recommend using [JHbuild](https://developer.gnome.org/jhbuild/stable/) to build this code, especially if you are planning to make changes to it.
```

Comment:
There are a lot of inline links; I've found that Markdown is a little bit more readable if you use references, and put the links at the bottom. It also helps in keeping the line length to a comfortable 80 columns.